mlflow/pipelines/regression/v1/resources/pipeline_dag_template.html

```diff
@@ -12,7 +12,7 @@
         display: flex;
       }
       .mermaid {
-        width: 60%;
+        width: 70%;
```

Comment:
@sunishsheth2009 I changed this CSS value because the graph was getting too small to see the text clearly. Wondering if we should instead change the editor to show up below the pipeline graph, so that the graph itself can have sufficient space.

Reply:
I think the problem is, that it would be a long scroll for users to see the corresponding click action and for smaller screens that might not be a good idea IMO. What do you think?

Reply:
Per offline discussion, we decided to rearrange the graph itself.